catroid/src/main/java/org/catrobat/catroid/common/Constants.java

```diff
@@ -111,6 +111,8 @@
 	public static final String CATROBAT_TOKEN_LOGIN_URL = BASE_URL_HTTPS + "tokenlogin?username=";
 	public static final String CATROBAT_DELETE_ACCOUNT_URL = BASE_URL_HTTPS + "profile/edit";
 
+	public static final String PLAY_STORE_PAGE_LINK = "https://play.google.com/store/apps/details?id=";
```

Comment:
I guess no need for this any more. right?

Reply:
I just added it. It's when you are asked to rate the app.

Reply:
actually I dont see any usage for it in your Pr, do you mean here `		Uri.parse("market://details?id=" + getPackageName())).addFlags(Intent.FLAG_ACTIVITY_NEW_TASK));`

Reply:
[here](https://github.com/Catrobat/Catroid/pull/3104/files#diff-283bd7edee90877049fecac924c059cfR332) it is!

Reply:
My bad, I used "find in page" to look for the usage, it seems broken in my browser. *_#